packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.Props.ts

```diff
@@ -52,4 +52,9 @@ export interface ISearchBoxProps extends React.Props<SearchBox> {
    * @defaultvalue labelText
    */
   ariaLabel?: string;
+
+  /**
+   * The boolean which indicates whether to hide search icon.
```

Comment:
I don't know that I agree with this fix. If you don't want the search icon, why not just use `TextField` instead?